cypress/integration/Tooltip.spec.js

```diff
@@ -0,0 +1,26 @@
+/// <reference types="Cypress" />
+
+/* eslint-disable no-undef */
+describe('The Tooltip component', () => {
+  beforeEach(() => {
+    cy.visit(
+      '/iframe.html?selectedKind=Overlays%7CTooltip&selectedStory=Default%20Style&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel',
+    );
+  });
+
+  it('should open when a mouse over event triggers.', () => {
+    cy.queryAllByText('Messages an ISP or other remote domain accepted').should('not.be.visible');
+    cy.get('button')
+      .first()
+      .trigger('mouseover');
+    cy.queryAllByText('Messages an ISP or other remote domain accepted').should('be.visible');
```

Comment:
Do we wanna test the mouseOut behavior as well?

Reply:
I'll play around with that event and see if it makes sense. Not sure currently.